store/sqlite/sqlite.go

```diff
@@ -405,12 +405,24 @@ func (d db) DeleteGuestLink(id types.GuestLinkID) error {
 	return tx.Commit()
 }
 
-// Purge clears orphaned rows from the database.
+// Purge deletes expired entries and clears orphaned rows from the database.
 func (d db) Purge() error {
+	log.Printf("deleting expired entries from database")
+
+	_, err := d.ctx.Exec(`
+	DELETE FROM
+		entries
+	WHERE
+		entries.expiration_time IS NOT NULL AND entries.expiration_time < CURRENT_TIMESTAMP;`)
```

Comment:
Instead of `CURRENT_TIMESTAMP`, can we format the timestamp using `formatTime`?

`expiration_time` and `CURRENT_TIMESTAMP` use two different formats:

```bash
$ sqlite3 data/store.db "SELECT entries.expiration_time, CURRENT_TIMESTAMP FROM entries"
2022-09-06T14:37:50Z|2022-08-22 23:57:35
2022-09-06T17:03:55Z|2022-08-22 23:57:35
```

Or, side by side:

```text
2022-09-06T14:37:50Z
2022-08-22 23:57:35
```

Reply:
Yes, you are right, expired entries with current date in timestamp are not deleted. I'll fix it.